tests/test_controller.py

```diff
@@ -1430,6 +1440,51 @@ def send(self, *args, **kwargs):
         assert response.uri == INTEGRATION_ERROR.uri
         assert response.detail == "SMTP error while sending email to mailto:help@library.org"
 
+    def test_registration_fails_if_email_server_unusable(self):
```

Comment:
Added a test for the missing or unresponsive mail server, as distinct from one raising an SMTP problem.